automated-testing/readme.md

```diff
@@ -1,25 +1,56 @@
+| Development; Staging; Operation | Discover overall system melt point (the loads at which the end-to-end system fails) and which component is the weakest link in the whole stack | Squeeze; [Load testing (stress)](performance-testing/load-testing.md) |
+| Development; Staging; Operation | Measure capacity limits for given provisioning to predict or satisfy future provisioning needs | Squeeze; [Load testing (stress)](performance-testing/load-testing.md) |
+| Development; Staging; Operation | Create / exercise failover runbook | Failover drills |
+| Development; Staging; Operation | Prove disaster recoverability – loss of data center (the meteor scenario); measure MTTR | DR drills |
+| Development; Staging; Operation | Understand whether observability dashboards are correct, and telemetry is complete; flowing | Trace Validation; [Load testing (stress)](performance-testing/load-testing.md); Scenario; [End-to-End testing](e2e-testing/readme.md) |
+| Development; Staging; Operation | Measure impact of seasonality of traffic | [Load testing](performance-testing/load-testing.md) |
+| Development; Staging; Operation | Prove Transaction and alerts correctly notify / take action | [Synthetic Transaction testing](synthetic-monitoring-tests/readme.md) (negative cases); [Load testing](performance-testing/load-testing.md) |
+| Development; Staging; Operation; Optimizing | Understand scalability curve, i.e. how the system consumes resources with load | [Load testing (stress)](performance-testing/load-testing.md); [Performance testing](performance-testing/readme.md) |
+| Operation; Optimizing | Discover system behavior over long-haul time | Soak |
+| Optimizing | Find cost savings opportunities | Squeeze |
+| Staging; Operation | Measure impact of failover / scale-out (repartitioning, increasing provisioning) / scale-down | Failover drills; Scale drills |
+| Staging; Operation | Create/Exercise runbook for increasing/reducing provisioning | Scale drills |
+| Staging; Operation | Measure behavior under rapid changes in traffic | Spike |
+| Staging; Optimizing | Discover cost metrics per unit load volume (what factors influence cost at what load points, e.g. cost per million concurrent users) | Load (stress) |
+| Development; Operation | Discover points where a system is not resilient to unpredictable yet inevitable failures (network outage, hardware failure, VM host servicing, rack/switch failures, random acts of the Malevolent Divine, solar flares, sharks that eat undersea cable relays, cosmic radiation, power outages, renegade backhoe operators, wolves chewing on junction boxes, …) | Chaos |
 
 ## Sections within Testing
 
 - [End-to-End testing](e2e-testing/readme.md)
 - [Integration testing](integration-testing/readme.md)
 - [Performance testing](performance-testing/readme.md)
-- [Synthetic Monitoring testing](synthetic-monitoring-tests/readme.md)
+- [Synthetic Transaction testing](synthetic-Monitoring-tests/readme.md)
```

Comment:
I find this change to be a bit misleading, since none of the referenced document's titles have been changed accordingly.

Reply:
1. yes, you are correct and that is due to a mistake on my part.  Lesson learned.
2. a. consistency will arrive over time.  b. I think the template is easy to follow.